package.json

```diff
@@ -14,10 +14,10 @@
     "scripts": {
         "start": "node run.js",
         "test": "jest",
-        "test-coverage": "jest --coverage --collectCoverageFrom='src/*'",
+        "test-coverage": "jest --coverage --collectCoverageFrom='src/**'",
         "lint": "eslint *.js src tests",
         "lint-json": "find . -name '*.json' -not -path './node_modules/*' -print0 | xargs -t -n1 -0 jsonlint -q",
-        "format": "./node_modules/.bin/prettier --write --require-pragma *.js src/*.js tests/*.js tests/macros/*.js"
+        "format": "./node_modules/.bin/prettier --write --require-pragma *.js src/*.js src/**/*.js tests/*.js tests/**/*.js"
```

Comment:
The `**` thing isn't working quite right: this now omits src/*.js and tests/*.js. I've added my own mini commit to change htat.